plans/elixir/habitat/plan.sh

```diff
@@ -0,0 +1,46 @@
+pkg_origin=mozillareality
+pkg_name=elixir
+pkg_version=1.14.3
+pkg_description="A dynamic, functional language designed for building scalable and maintainable applications. Elixir leverages the Erlang VM, known for running low-latency, distributed and fault-tolerant systems, while also being successfully used in web development and the embedded software domain."
+pkg_upstream_url=http://elixir-lang.org
+pkg_maintainer="Mozilla Mixed Reality <mixreality@mozilla.com>"
+pkg_license=('Apache-2.0')
+pkg_source="https://github.com/elixir-lang/elixir/archive/v${pkg_version}.tar.gz"
+pkg_shasum=bd464145257f36bd64f7ba8bed93b6499c50571b415c491b20267d27d7035707
+pkg_deps=(
+  core/busybox
+  core/cacerts
+  core/coreutils
+  core/openssl
+  core/erlang/23.3.4.9
```

Comment:
If I recall correctly, we were confused why we needed this particular version of `erlang`, and why we couldn't use `mozillareality/erlang/23.3.4.18` here. _We can_. 
I think the problem we were running into before is that we uploaded `mozillareality/erlang/23.3.4.18` to `bldr.reticulum.io` but were trying to _build_ reticulum with the packages on `bldr.habitat.sh` (which did not include that version).

Reply:
I was able to build this plan with `bldr.habitat.sh` as the origin but not with `bldr.reticulum.io` as the origin. This even after uploading all the core packages to `bldr.reticulum.io`. I’m sure there’s a way to do it, but I was going for the simplest thing that could possibly work.